samples/python/winml_tutorial/complete_winml_tutorial.py

```diff
@@ -24,92 +29,68 @@ def sync_wrapper(*args, **kwds):
         ret = await fun(*args, **kwds)
 
         end = time.perf_counter()
-        print(fun.__name__, "took", end - start, "seconds")
+        print(f"{fun.__name__} took {end - start:.3f} seconds")
         return ret
 
     return async_wrapper if asyncio.iscoroutinefunction(fun) else sync_wrapper
 
-import pyrt.windows.ai.machinelearning as winml 
-
 @timed_op
 def load_model(model_path):
-    return winml.LearningModel.LoadFromFilePath(model_path)
+    return winml.LearningModel.load_from_file_path(os.fspath(model_path))
```

Comment:
Currently, this tutorial is pinned against an older version of the pywinrt projection that doesn't have pythonic member names. How about you make these changes to src\test\python\complete_winml_tutorial.py and I'll update samples\python\winml_tutorial\complete_winml_tutorial.py as part of the overall effort to update samples\python\winml_tutorial